powqutyd/files/src/uci_config.c

```diff
@@ -28,6 +28,25 @@ static long uci_lookup_option_long(struct uci_context *uci,
 	return str == NULL ? -1 : atol(str);
 }
 
+/*
+ * check option value from uci config
+ * @param option_value: name of the option to check
+ * @param option_length: max length of option value
+ * @return 1 on to long option value, -1 if unset, 0 else
+ */
+static int check_option_value(char * option_value, unsigned int option_length) {
```

Comment:
should we change the function name  to be more clear ?
I would prefer something like: 
  static bool config_option_is_valid() ... 0 = failure, non-zero = success